### PR TITLE
[IMP] agreement_maintenance: Link Equipments and Service Profiles

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by max3903** [comment]
Ping @barnarj @kvedder-amplex

**Comment by kvedder-amplex** [comment]
@max3903 Looks good!

File: agreement_maintenance/views/maintenance_equipment_view.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            </field>
+            <xpath expr="//notebook" position="inside">
+                <page string="Services" id="services">
+                    <field name="serviceprofile_ids"
+                           domain="[('active', '=', True),
+                                    ('equipment_id', '=', False)]"
+                           widget="many2many">
+                        <tree>
+                            <field name="name"/>
+                            <field name="partner_id"/>
```
Comment (patrickrwilson) [inline agreement_maintenance/views/maintenance_equipment_view.xml:22]: When I try to install the module on a clean v12 instance, I receive an error stating: Field `partner_id` does not exist...
  File "C:\Program Files (x86)\Odoo 12.0e\server\odoo\models.py", line 1112, in _validate_fields
    raise ValidationError("%s\n\n%s" % (_("Error while validating constraint"), tools.ustr(e)))
odoo.tools.convert.ParseError: "Error while validating constraint

Field `partner_id` does not exist
Error context:
View `maintenance.equipment.form.agreement`
[view_id: 525, xml_id: n/a, model: maintenance.equipment, parent_id: 464]
None" while parsing file:/c:/program%20files%20(x86)/odoo%2012.0e/server/odoo/addons/agreement_maintenance/views/maintenance_equipment_view.xml:6, near
<record id="maintenance_equipment_form_view_agreement" model="ir.ui.view">

**Comment by max3903** [comment]
@patrickrwilson I just rebased this branch. It should include #368 and works now.

**Review by patrickrwilson — APPROVED** [review]
Looks good

File: agreement_maintenance/views/maintenance_equipment_view.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <field name="inherit_id"
+               ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <field name="owner_user_id" position="after">
+                <field name="agreement_id"/>
+            </field>
+            <xpath expr="//notebook" position="inside">
+                <page string="Services" id="services">
+                    <field name="serviceprofile_ids"
+                           domain="[('active', '=', True),
```
Comment (bodedra) [inline agreement_maintenance/views/maintenance_equipment_view.xml]: Doesn't Odoo by default fetch only active records ?

```suggestion
                           domain="[('equipment_id', '=', False)]"
```

File: agreement_maintenance/views/maintenance_equipment_view.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                                    ('equipment_id', '=', False)]"
```
Comment (bodedra) [inline agreement_maintenance/views/maintenance_equipment_view.xml]:
```suggestion
```

**Review by bodedra — APPROVED** [review]
LGTM :+1:  Added minor comments.

**Pull request merged**